m4_sessions/1810-inning/problems_tech_cannot_solve.md

```diff
@@ -18,14 +17,25 @@
      All sessions NEED to have a closing activity, reflection, discussion
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+  1. intro to topic by John
+  1. we will split into two seperate groups. One group will be for e-scooters and one will be against.
+    * starter points for debate
+    1. e-waste
+    1. acces to transportation for low cost
+    1. safety
+    1. maintenance
+    1. lack of public transportation in city
 
-## Recommended Trajectory 
+  1. Debate topic (simple/see every day) : e-scooters/dockless bikes -> https://www.denverpost.com/2018/10/01/denver-dockless-scooters-confusion-injuries/
+  1. secondary(backup): chris presents this topic: https://www.economist.com/international/2017/11/23/much-of-the-money-donated-after-disasters-is-stolen
+    * how tech helped stop corruption in disaster relief donations
 
-    ## Introductons 
-    ## Intro to new material
-    ## Small Group Activity
-    ## Closing Discussion
 
+## Recommended Trajectory 
+  *  Introductons 10mins
+  *  Intro to debate format (rearrange tables if needed) 10 mins
+  *  Debate/discussion - each side shares their arguments (we help prompt if needed) 20mins 
+  *  Closing Discussion as a group to review key takeaways.  We all talk about key take aways. 15 mins. 
+  *  concluding video: https://www.ted.com/talks/jason_pontin_can_technology_solve_our_big_problems?language=en
```

Comment:
This is great.  I would LOVE for you to have a 5 min turn and talk after this to give people time to process it.